com/williamfiset/algorithms/search/InterpolationSearch.java

```diff
@@ -14,7 +14,7 @@
    * @param nums - an ordered list containing uniformly distributed values.
    * @param val - the value we're looking for in 'nums'
    */
-  static int interpolationSearch(int[] nums, int val) {
+  public static int interpolationSearch(int[] nums, int val) {
```

Comment:
Thanks, I was just about to mention that this should be public to be visible outside the package. I think a lot of untested methods are like this.